utils/constants.js

```diff
@@ -127,6 +127,8 @@ export const FEATURED_SERVICE_PATH = '/requests/new'
 // the default is 1 week
 export const EXPIRATION_DURATION = 604800000
 
+export const API_PER_PAGE = 2000
```

Comment:
Do you actually load them all on the page at once? Or is there some kind of infinite scroll or pagination?

Reply:
it's an infinite scroll. there's a video in the pr comment if that helps. doesn't seem to be problematic. but also....the beachsidebiotech and phenovista marketplaces each have under 600 wares. I went with 2000 for insurance with other marketplaces.

and in the next phase, or with the next client, we may introduce pagination.